samples/bluetooth/peripheral/src/main.c

```diff
@@ -323,11 +320,12 @@ void main(void)
 {
 	int err;
 
-	err = bt_enable(bt_ready);
+	err = bt_enable();
```

Comment:
This can't possibly compile since you're not changing the bt_enable API - it still has one parameter. I guess you intended to pass NULL here? Please always at least compile-test your changes 😉

Reply:
Sorry, I was being lazy and letting CI do it for me.